### PR TITLE
chore(infra): move to internal apps infra

### PR DESCRIPTION
**Pull request by polenz**
related to #IN-270

### CODE REVIEW

File: terraform/application/cloud_run.tf [hunk 1]
```diff
@@ -0,0 +1,58 @@
+locals {
+  image_url = "${var.region}-docker.pkg.dev/${var.google_project_id}/${google_artifact_registry_repository.hapdfy.repository_id}/hapdfy@${var.image_digest}"
+}
+
+resource "google_cloud_run_service" "hapdfy" {
+  depends_on = [
+    google_project_iam_member.hapdfy
+  ]
+  name     = "hapdfy"
```
Comment (vringar) [inline terraform/application/cloud_run.tf:9]:
```suggestion
  name     = "{var.prefix}hapdfy"
```
As both stage and live run in the same project all named resources need to have the prefix applied

File: terraform/application/sa-cloud-run.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+resource "google_service_account" "hapdfy" {
+  account_id   = "hapdfy-sa"
```
Comment (vringar) [inline terraform/application/sa-cloud-run.tf:2]:
```suggestion
  account_id   = "{var.prefix}hapdfy-sa"
```

**Review by vringar — CHANGES_REQUESTED** [review]
A couple of notes but overall this looks great!

File: .github/workflows/deployer.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+name: "Deployment"
+on:
+  push:
+    branches: [master, main, live]
+  pull_request:
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+env:
+  PERFORM_CHANGES: ${{(github.ref == 'refs/heads/master' || github.ref == 'refs/heads/live') && github.event_name == 'push'}}
```
[the remark this answers is not in the document]
Comment (polenz) [reply .github/workflows/deployer.yml:12]: yes, that's right.

**Review by vringar — APPROVED** [review]

**Pull request merged**